frontend/app/src/next/components/ui/docs-button.tsx

```diff
@@ -32,10 +33,12 @@ export function DocsButton({
   ...props
 }: DocsButtonProps) {
   const { open } = useDocs();
+  const { close: closeSideSheet } = useSideSheet();
 
   const handleClick = (e: React.MouseEvent) => {
     if (method === 'sheet') {
       e.preventDefault();
+      closeSideSheet();
```

Comment:
🤔 not sure where we should put this, but i think maybe it should be in open instead -- we're not directly using this button in the sidebar docs button for example

Reply:
maybe we should just merge these into one state and have the docs component rendered as a sidebar state?

im cool with the "one sidebar" design concept but i think it's going to get hairy to have to open and close each thing

Reply:
Very much agree - I think it'd be much easier with one component